ReactApp1/ReactApp1.Server/Controllers/Base/LoginController.cs

```diff
@@ -0,0 +1,93 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using ReactApp1.Server.Data;
+using ReactApp1.Server.Models;
+using ReactApp1.Server.Models.Models.Base;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.IdentityModel.Tokens;
+
+namespace ReactApp1.Server.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class LoginController : ControllerBase
+    {
+        private readonly ILogger<RegisterController> _logger;
+        private readonly JwtSettings _jwtSettings;
+        private readonly AppDbContext _context;
```

Comment:
Manau dabar būtų galima _context perkelt į repository. Ir čia naudoti service'ą.

Reply:
Aha, šitą galima bus padaryt. Bet galvoju tada, kai sumerginsim abu branchus